src/inputs/SelectField.mock.tsx

```diff
@@ -15,6 +15,7 @@ export function SelectField<T extends object, V extends Key>(props: SelectFieldP
     onBlur,
     onFocus,
     disabled,
+    disabledOptions,
```

Comment:
**suggestion:** Rather than having to check and see if the array is defined for each options as we loop through, we could default it to an empty array.
```
const { disabledOptions = [] } = props;

...
options.map((o, i) => {
  return <option disabled={disabledOptions.includes(getOptionValue(option).toString())}>
})
```

Reply:
I was thinking about implementing with a fallback empty array so I'll make that update. Cheers